### PR TITLE
COR-412 Core admin Dockerfile

### PR DESCRIPTION
**Pull request by ludydoo**

### CODE REVIEW

File: configs/authnz-frontend/nginx.conf.template [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  index index.html;
+
+  location / {
+       try_files $uri $uri/ /index.html;
+  }
+
+  location ~ $/(static)/ {
+    gzip on;
+    gzip_http_version  1.0;
```
Comment (neb42) [inline configs/authnz-frontend/nginx.conf.template:16]: I'm not sure what difference this would have, but why isn't this `1.1`?

Comment (ludydoo) [reply configs/authnz-frontend/nginx.conf.template:16]: Will fix. Seems 1.1 is the default, so I'll remove this stanza

**Comment by neb42** [comment]
Looks good for now, but we may want to move the package building out of here and into a build/publish to npm pipeline. This can then just install/build the react app.

**Pull request merged**